devtools/conda-envs/basic.yaml

```diff
@@ -39,7 +39,7 @@ dependencies:
 
   - openff-toolkit
   - openff-units >=0.2.1
-  - pydantic
+  - pydantic >1.10.17,<3
```

Comment:
This is probably my packaging naivete showing, but is there a reason not to do `>=2`? I see a version 1.10.18 in the pydantic docs, which would be allowed by these constraints right? Maybe it also has the v1 module.